README.md

```diff
@@ -18,16 +18,19 @@
 5. have bot listing requirements (such as no NSFW outside NSFW channels, respect rate-limits, don't respond to other bots, etc.)
 6. blur NSFW profile pictures.
 7. have a ToS and Privacy page.
+8. have a notice in footer or anywhere easily reachable, stating that your site isn't affiliated with discord.
+9. have support for slash bots (like letting prefix to be not set, let users know whether this bot is slash command only or slash + bot, or bot only, etc.)
+10. not serve broken images, maybe let the broken images get replaced with custom image.
 
 ---
 
 *Bot lists should meet these requirements to be a good bot list.*
 
 ## Your bot list should:
 
-1. not display all bots on the homepage.
+1. not display all bots on the homepage, the footer should be reachable without loading more and more bots.
```

Comment:
Maybe change to "no infinite scrolling"?

Reply:
Ok sure